aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -119,7 +119,7 @@ module aptos_std::simple_map {
         let right = length;
 
         while (left != right) {
-            let mid = (left + right) / 2;
```

Comment:
This is probably unnecessary. I don't think it's possible to overflow on this. This would require the vector to contain more than u64.max / 2. A vector that big will never be loadable.

Reply:
Yeah, I just wanna clean up any bad practice in coding.

Reply:
left + (right - left) / 2 costs more gas esp because it's in a loop....

Reply:
I can change it tmr to use the same gas :)